app/presenters/scope_filters_presenter.rb

```diff
@@ -1,29 +1,42 @@
 require 'gds_api/support_api'
 
 class ScopeFiltersPresenter
-  attr_reader :path, :organisation_slug
+  attr_reader :paths, :organisation_slug, :paths_for_api
 
-  def initialize(path: nil, organisation_slug: nil)
-    @path = normalize_path(path)
+  def initialize(paths: nil, organisation_slug: nil)
+    @paths = normalize_paths(paths)
     @organisation_slug = organisation_slug
   end
 
   def filtered?
-    path.present? || organisation_slug.present?
+    paths.present? || organisation_slug.present?
   end
 
   def invalid_filter?
     !filtered?
   end
 
   def done_page?
-    path.present? ? path.start_with?("done", "/done") : false
+    paths.present? ? paths.split(', ').any? { |path| path.start_with?("done", "/done") } : false
```

Comment:
We do `paths.split(', ')` a few times, maybe we should use `paths_for_api` because it's already an array?  We might want to think up a better name for that though, because `paths_for_api` makes sense in the place we use it outside the class, but not really inside.... 🤔

Reply:
`parsed_for_api` is now only used outside the class